spring-session/src/main/java/org/springframework/session/MapSessionRepository.java

```diff
@@ -34,6 +34,7 @@
  * </p>
  *
  * @author Rob Winch
+ * @author Eddú Meléndez
  * @since 1.0
  */
 public class MapSessionRepository implements SessionRepository<ExpiringSession> {
```

Comment:
Maybe we should use `MapSession` in the generic type like in other repositories?
@rwinch Is this breaking change?